### ISSUE TITLE
Incorrect parsing of element access for slice fields.

### ISSUE THREAD

**Issue by MartinClochard** [viperproject/gobra#173]
Gobra cannot parse correctly expressions `s.f[i]`. Those currently requires parenthesis around the field access to be accepted by Gobra. Go itself does not have such restrictions.

Example:
```
package pkg

type sliceStruct struct {
  f []int
}

requires forall i int :: 0 <= i && i < len(s.f) ==> acc(&s.f[i],_)
requires 0 <= j && j < len(s.f)
func test(s sliceStruct,j int) int {
  return s.f[j]
}
```
is rejected with a parsing error, unless both `s.f[_]` are replaced by `(s.f)[_]`

**Comment by ArquintL** [comment]
Related to #145

**Comment by jcp19** [comment]
Seems to have been fixed in #388
